package.json

```diff
@@ -34,11 +34,12 @@
     "file-saver": "^2.0.5",
     "history": "^4.10.1",
     "jszip": "^3.8.0",
+    "lodash": "^4.17.21",
```

Comment:
Why lodash?

Reply:
Used it to merge different styles , but removed and used spread operator